app/models/state_file_dependent.rb

```diff
@@ -55,6 +59,10 @@ class StateFileDependent < ApplicationRecord
   enum eic_disability: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eic_disability
   enum eic_student: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eic_student
   enum id_has_grocery_credit_ineligible_months: { unfilled: 0, yes: 1, no: 2 }, _prefix: :id_has_grocery_credit_ineligible_months
+  enum nj_dependent_attends_accredited_program: { unfilled: 0, yes: 1, no: 2 }, _prefix: :nj_dependent_attends_accredited_program
+  enum nj_dependent_enrolled_full_time: { unfilled: 0, yes: 1, no: 2 }, _prefix: :nj_dependent_enrolled_full_time
+  enum nj_dependent_five_months_in_college: { unfilled: 0, yes: 1, no: 2 }, _prefix: :nj_dependent_five_months_in_college
+  enum nj_filer_pays_tuition_for_dependent: { unfilled: 0, yes: 1, no: 2 }, _prefix: :nj_filer_pays_tuition_for_dependent
```

Comment:
[dust] i think since these ultimately end up decided by checkboxes that only have the checked/unchecked state as yes/no, something feels odd to me about using the same enums as we do for radio buttons where the user can say yes, or no, or leave unfilled.

But since the answer implicitly becomes `no` if a user goes through the form without clicking the box, I guess it's ultimately a UX question -- if we're ok not being able to have the same validations as we do on an unfilled radio button, then i guess having three states for this is fine after all. But just wanted to mention because it's the first time I'm encountering this in our UX.

Reply:
Agree that checkboxes only have filled and unfilled states and we might want to figure out if there's a better enum

Reply:
This is a good point, can we discuss what it *should* be?

Also, I'm thinking I might like to resolve this in a follow-up refactor because we should also change the other checkboxes we have that following the same pattern, which I believe would be the `homeowner_*` and `tenant_*` ones

Reply:
recap of discussion is that:
* we'll verify that submitting the form with an unchecked box indeed turns this value to `no`
* keep the word `unfilled` to match other values... but to avoid confusion with an unfilled checkbox we can add comments and group enums for checkboxes vs radio buttons separately in the code